docs/customizing.rst

```diff
@@ -213,6 +213,36 @@ Celery. The practical way with custom serialization may look like so::
 .. _Celery: http://www.celeryproject.org/
 
 
+Custom send_mail method
+-----------------------
+
+It's also possible to completely override the ``security.send_mail`` method to
+implement your own logic, like so:
+
+    from flask import Flask
+    from flask_security import Security, SQLAlchemyUserDatastore
+
+    security = Security()
+
+    def create_app(config):
+        """Initialize Flask instance."""
+
+        app = Flask(__name__)
+        app.config.from_object(config)
+
+        def custom_send_mail(subject, recipient, template, **context):
+            # implement your own logic here
```

Comment:
can you add an example when it would be necessary to override the `send_mail` method?